mutt/buffer.c

```diff
@@ -453,3 +453,20 @@ size_t mutt_buffer_copy(struct Buffer *dst, const struct Buffer *src)
 
   return mutt_buffer_addstr_n(dst, src->data, mutt_buffer_len(src));
 }
+
+
+/** 
+ * mutt_buffer_seek - set current read/write position to offset from beginning
+ * @param buffer Buffer to use
+ * @param offset size position from beginning to set
```

Comment:
"size position" reads weird

Reply:
I was using the "@param {{param_name}} {{param_type}} {{explanation}}" pattern

I am open to suggestions how to improve this. English is not my best skill :)